GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeViewModel.swift

```diff
@@ -285,6 +285,21 @@ extension SecureConversations.WelcomeViewModel {
 
         return text.count <= Self.messageTextLimit
     }
+
+    static func headerState(
+        for instance: SecureConversations.WelcomeViewModel
+    ) -> Header.Props {
+
+        .init(
+            title: instance.environment.welcomeStyle.headerTitle,
+            effect: .none,
+            endButton: .init(),
+            backButton: .init(tap: Cmd { [weak instance] in instance?.delegate?(.backTapped) }),
```

Comment:
Is it possible to form props for buttons on Header.Props level?

Reply:
not always, some places assign action for back button directly. in future perhaps we can change it, after EngagementViewController refactoring.